web/styles/cm-hints.scss

```diff
@@ -0,0 +1,17 @@
+@import 'package:dart_pad/scss/colors';
```

Comment:
Is this import needed? It doesn't seem to be used.

Reply:
No it isn't needed. I guess I had added it for consistency. All of the other cm-*.scss files have the same import even when it isn't used.